README.md

```diff
@@ -65,10 +65,10 @@ rustup install nightly
 * OpenCL bindings
 
 ```shell
-# Ubuntu
+# Ubuntu (20.04)
 sudo make install-deps
 # Or
-sudo apt install build-essential clang ocl-icd-opencl-dev libssl-dev
+sudo apt-get install --no-install-recommends -y build-essential clang ocl-icd-opencl-dev aria2 cmake
```

Comment:
Let's stick to `apt`. And `aria2` is an optional dependency.

Reply:
Fixed